msdb/msdb/templates/search/search.html

```diff
@@ -24,11 +29,11 @@ <h5>{{ movie.title }}</h5>
     </div>
     <div class="col-sm-6 col-md-3">
       <h2>People</h2>
-      {% for i in '0123456789kkkkk'|make_list %}
+      {% for person in people %}
         <a href="#" class="search_link">
           <div class="mb-4">
-            <h5 style="margin-bottom: 0;">Emma Stone</h5>
-            <p>Actor/Actress</p>
+            <h5 style="margin-bottom: 0;">{{ person.name }}</h5>
+            <p> {% if person.director %}Director {% endif %} {% if person.actor %}Actor/Actress{% endif %}</p>
```

Comment:
add comma between Director and Actor/Actress

Reply:
Fixed